exporter/exporterhelper/obsreport.go

```diff
@@ -55,6 +56,12 @@ func newInstruments(registry *metric.Registry) *instruments {
 		metric.WithLabelKeys(obsmetrics.ExporterKey),
 		metric.WithUnit(metricdata.UnitDimensionless))
 
+	insts.queueCapacity, _ = registry.AddInt64DerivedGauge(
+		obsmetrics.ExporterKey+"/queue_capacity",
+		metric.WithDescription("Current capacity of the retry queue (in batches)"),
```

Comment:
I feel like "Current" implies the capacity can change at some point, but until the collector is restarted it is a fixed size right?  Maybe something like "Fixed capacity..." is clearer?

Reply:
You are right, I will fix this